docs/installation.md

```diff
@@ -64,40 +67,63 @@ You may have to set your specific configurations if it is deployed into a produc
+| `daemon.extraEnvs`                        | Extra environment variables that will be pass onto pods      | `[]`                          |
 | `daemon.log.level`                        | Log level that kruise-daemon printed                         | `4`                           |
 | `daemon.port`                             | Port of metrics and healthz that kruise-daemon served        | `10221`                       |
+| `daemon.pprofAddr`                        | Address of pprof served                                      | `localhost:10222`             |
 | `daemon.resources.limits.cpu`             | CPU resource limit of kruise-daemon container                | `50m`                         |
 | `daemon.resources.limits.memory`          | Memory resource limit of kruise-daemon container             | `128Mi`                       |
 | `daemon.resources.requests.cpu`           | CPU resource request of kruise-daemon container              | `0`                           |
 | `daemon.resources.requests.memory`        | Memory resource request of kruise-daemon container           | `0`                           |
 | `daemon.affinity`                         | Affinity policy for kruise-daemon pod                        | `{}`                          |
 | `daemon.socketLocation`                   | Location of the container manager control socket             | `/var/run`                    |
 | `daemon.socketFile`                       | Specify the socket file name in `socketLocation` (if you are not using containerd/docker/pouch/cri-o) | ` ` |
-| `webhookConfiguration.failurePolicy.pods` | The failurePolicy for pods in mutating webhook configuration | `Ignore`                      |
-| `webhookConfiguration.timeoutSeconds`     | The timeoutSeconds for all webhook configuration             | `30`                          |
-| `crds.managed`                            | Kruise will not install CRDs with chart if this is false     | `true`                        |
-| `manager.resyncPeriod`                    | Resync period of informer kruise-manager, defaults no resync | `0`                           |
-| `manager.hostNetwork`                     | Whether kruise-manager pod should run with hostnetwork       | `false`                       |
-| `imagePullSecrets`                        | The list of image pull secrets for kruise image              | `false`                       |
+| `daemon.credentialProvider.enable`        | Whether to enable credential provider for image pull job     | `false`                       |
+| `daemon.credentialProvider.hostPath`      | credential provider plugin node dir, will volume mount into kruise-daemon | `credential-provider-plugin` |
+| `daemon.credentialProvider.configmap`     | credential provider yaml configmap name in kruise-system ns  | `credential-provider-config`  |
```

Comment:
`credential provider plugin node dir, will volume mount into kruise-daemon` -> `node dir of the credential provider plugin, kruise-daemon will mount the dir as a hostpath volume

`credential provider yaml configmap name in kruise-system ns ` -> `configmap name of the credential provider in kruise-system ns`